File: python/packages/varar/src/varar/registry.py
"""Adapter-only glue (mirrors @varar/varar/registry): build the registry and
context factory from the module-scope accumulator, and reset it between runs."""

from varar.internal import _custom_parameter_types, _reset_builder, build_registry, context_factory

__all__ = ["build_registry", "context_factory", "_reset_builder", "_custom_parameter_types"]

File: ruby/packages/varar/lib/varar/registry.rb
# frozen_string_literal: true

require 'varar/internal'

module Varar
  # Adapter/harness glue — mirrors the `@varar/varar/registry` subpath.
  # Authors import only `steps` (via varar); runners and the conformance
  # harness reach the accumulator through here.
  module RegistryGlue
    module_function

    def reset_builder
      Internal.reset_builder
    end

    def build_registry
      Internal.build_registry
    end

    def context_factory
      Internal.context_factory
    end

    def custom_parameter_types
      Internal.custom_parameter_types
    end
  end
end
